tools/python/boutiques/bosh.py

```diff
@@ -20,6 +20,19 @@
 from tabulate import tabulate
 
 
+def prettyprint(*params):
+    parser = ArgumentParser("Boutiques pretty-print for generating help text")
+    parser.add_argument("descriptor", action="store",
+                        help="The Boutiques descriptor.")
+    results = parser.parse_args(params)
+
+    from boutiques.prettyprint import pprint
+    with open(results.descriptor, 'r') as fhandle:
```

Comment:
To make pretty-print compatible with JSON objects and Zenodo files, you can do something like:
```
desc = loadJson(results.descriptor)
helpstring = pprint(desc)
```

Reply:
Great catch - thanks!